sqlite-db/src/models.rs

```diff
@@ -749,18 +762,68 @@ impl From<model::olivia::BitMexPriceEventId> for BitMexPriceEventId {
         Self {
             timestamp: id.timestamp(),
             digits: id.digits(),
+            index: id.index_price().into(),
         }
     }
 }
 
 impl From<BitMexPriceEventId> for model::olivia::BitMexPriceEventId {
     fn from(id: BitMexPriceEventId) -> Self {
-        model::olivia::BitMexPriceEventId::new(id.timestamp, id.digits)
+        model::olivia::BitMexPriceEventId::new(id.timestamp, id.digits, id.index)
     }
 }
 
 impl_sqlx_type_display_from_str!(BitMexPriceEventId);
 
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash, PartialOrd, Ord)]
+enum IndexPrice {
+    Bxbt,
+    Beth,
+}
+
+impl fmt::Display for IndexPrice {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        let s = match self {
+            IndexPrice::Bxbt => "BXBT",
+            IndexPrice::Beth => "BETH",
+        };
+
+        s.fmt(f)
+    }
+}
+
+impl FromStr for IndexPrice {
+    type Err = anyhow::Error;
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        Ok(match s {
+            "BXBT" => Self::Bxbt,
+            "BETH" => Self::Beth,
+            _ => bail!("Index price {s} not supported"),
+        })
+    }
+}
+
```

Comment:
I think you can ged rid of these manual impls with:
`#[strum(serialize_all = "UPPERCASE")]` 

(used elsewhere).

Reply:
Let's do it with https://github.com/itchysats/itchysats/issues/2751, because it's surprisingly annoying to do this given that we temporarily _serialize_ to the same string for both.

I'll add it to the issue.